src/components/App.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+
+const ListItem = (props) => {
+  return (
+    <li onClick={props.onClick}>
+      {props.item}
+    </li>
+  );
+};
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      list: [] // Need to push items into this array.
+    };
+  }
+  addItem() {
+    const item = document.getElementById('listItem').value;
+    document.getElementById('listItem').value = '';
```

Comment:
There are a lot of better ways to get things done in React than this. I would recommend looking at how to do things without having to select the 'document'